geomesa-spark/geomesa-spark-converter/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>geomesa-spark_2.11</artifactId>
+        <groupId>org.locationtech.geomesa</groupId>
+        <version>1.3.0-m3-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>GeoMesa Spark Converter RDD Provider</name>
+    <artifactId>geomesa-spark-converter_2.11</artifactId>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <!-- Hadoop FileInputFormat requires older guava version -->
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>12.0.1</version>
```

Comment:
Meh, let's forget about it.  Didn't know if there was a Hadoop-managed-Guava-version variable somewhere.